src/models/BrowserStyleSheet.js

```diff
@@ -55,13 +64,32 @@ class BrowserTag implements Tag {
     }
 
     const comp = { componentId, textNode: document.createTextNode('') }
-    this.el.appendChild(comp.textNode)
+    if (!USE_SPEEDY) {
+      this.el.appendChild(comp.textNode)
+    }
 
     this.size += 1
     this.components[componentId] = comp
   }
 
-  inject(componentId: string, css: string, name: ?string) {
+  speedyInsert(el: HTMLStyleElement, cssRules: Array<string>) {
+    const sheet = el.sheet
+    if (sheet === null || sheet === undefined) {
+      return
+    }
+
+    for (let i = 0; i < cssRules.length; i += 1) {
+      const rule = cssRules[i]
+      if (rule !== undefined && rule.length) {
+        /* eslint-disable */
+        // $FlowFixMe Flow's `StyleSheet` breakdown here https://github.com/facebook/flow/issues/2696
+        sheet.insertRule(rule, sheet.cssRules.length)
```

Comment:
do we no longer need to wrap this in a try/catch?

Reply:
Readded it in the latest commit; not sure whether a try/catch was there in the first place in this branch